Dockerfile

```diff
@@ -38,5 +38,7 @@ RUN gem install -N conjur-debify-${VERSION}.gem
 
 ARG CONJUR_APPLIANCE_URL
 ENV CONJUR_APPLIANCE_URL ${CONJUR_APPLIANCE_URL:-https://conjur-master-v2.itp.conjur.net/api}
+ENV CONJUR_ACCOUNT ${CONJUR_ACCOUNT:-conjur}
+ENV CONJUR_VERSION ${CONJUR_VERSION:-4}
```

Comment:
Why default to 4? Do we not use `debify` w/ Conjur v5?

Reply:
This is setting the version used to communicate with the appliance. By default, it talks to conjurops, which is still running v4.